.github/workflows/dev-release.yml

```
@@ -55,7 +55,7 @@ jobs:
       - name: Set up Go 1.24.2
         uses: actions/setup-go@v4
```

actionlint (1.7.7)

the runner of "actions/setup-go@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)